### ISSUE TITLE
How to build and deploy

### ISSUE THREAD

**Issue by Mavis2103** [Psychedelic/terabethia#74]
Hello, I want to build and deploy this project, but I don't know modify what to build success. Can you help me? Thank you.

**Comment by b0xtch** [comment]
> Hello, I want to build and deploy this project, but I don't know modify what to build success. Can you help me? Thank you.

hey you can check these docs, https://docs.terabethia.ooo/

**Comment by Mavis2103** [comment]
Yes, I know. But I don't know modify another address on source code or canister id. Can you help me?
